samples/subsys/display/lvgl/src/main.c

```diff
@@ -36,12 +33,26 @@ static void button_isr_callback(const struct device *port,
 }
 #endif
 
+#if IS_ENABLED(CONFIG_LV_Z_POINTER_KSCAN) || IS_ENABLED(CONFIG_LV_Z_POINTER_INPUT)
```

Comment:
Nit: Maybe add a macro for this to avoid long repititions.

Reply:
corrected